ilert_nagios.py

```diff
@@ -128,12 +133,15 @@ def main():
     parser.add_argument('-a', '--apikey', help='API key for the alert source in iLert')
     parser.add_argument('-e', '--endpoint', default='https://ilertnow.com',
                         help='iLert API endpoint (default: %(default)s)')
-    parser.add_argument('-p', '--port', type=int, default=443, help='endpoint port (default: %(default)s)')
+    parser.add_argument('-p', '--port', type=int, default=443, help='Endpoint port (default: %(default)s)')
     parser.add_argument('-d', '--dir', default='/tmp/ilert_nagios',
-                        help='event directory where events are stored (default: %(default)s)')
+                        help='Event directory where events are stored (default: %(default)s)')
     parser.add_argument('--version', action='version', version=PLUGIN_VERSION)
+    parser.add_argument('-k', '--insecure', type=bool, default=False,
+                        help='Allow insecure server connections when using SSL (default: %(default)s)')
```

Comment:
why not be more explicit, e.g. `Disable SSL certification validation (e.g. to use self-signed certificates)`?